shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -190,6 +190,16 @@ - (void)setupNotificationCenterObservers {
                object:nil];
 
   [center addObserver:self
+             selector:@selector(applicationDidEnterBackground:)
+                 name:UIApplicationDidEnterBackgroundNotification
```

Comment:
I am not sure how this will ever get to the framework. By the time the application has been backgrounded, the display links have been severed and there will be no more frame callbacks.